release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1724,6 +1732,12 @@ If these values are not defined in a failure domain configuration, for instance
 
 * Previously, entering an invalid value in any `cpuset` field under `spec.cpu` in the `PerformanceProfile` resource caused the webhook validation to crash. With this release, improved error handling for the `PerformanceProfile` validation webhook ensures that invalid values for these fields return an informative error. (link:https://issues.redhat.com/browse/OCPBUGS-45616[*OCPBUGS-45616*])
 
+* Previously, users could enter an invalid string for any CPU set in the performance profile, resulting in a broken cluster. With this release, the fix ensures that only valid strings can be entered, eliminating the risk of cluster breakage. (link:https://issues.redhat.com/browse/OCPBUGS-47678[*OCPBUGS-47678*])
+
+* Previously, configuring the Node Tuning Operator (NTO) using `PerformanceProfiles` created the `ocp-tuned-one-shot` `systemd` service, which ran before kubelet and blocked its execution. The `systemd` service invoked podman, which used the NTO image. When the NTO image was not present, podman tried to fetch the image. With this release, support for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env` are added. This support allows podman to pull the NTO image in environments that need to use `http(s)` proxy for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-39005[*OCPBUGS-39005*])
```

Comment:
Also, just checking, is NTO an approved acronym for the Node Tuning Operator?

Reply:
Thanks for your review @bergerhoffer! I corrected the content. As for `NTO`, Kevin Quinn used it recently for the Telco docs. I will check with him to see if he will add an attribute for this.